docs_nnx/guides/gemma.ipynb

```diff
@@ -124,7 +140,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Flax examples are not exposed as packages so you need to use the workaround in the next cells to import from NNX's Gemma example."
```

Comment:
@cgarciae Edited:

"To interact with the Gemma model, you will use the Flax NNX Gemma code from [`google/flax` examples on GitHub](https://github.com/google/flax.git). Since it is not exposed as packages, you need to use the following workaround in the next cells to import from the Flax NNX Gemma example."